system/views/admin/datamanager/groupVersionNavigator.cfm

```diff
@@ -0,0 +1,62 @@
+<cfscript>
```

Comment:
Should be renamed and moved to sys config views folder (see previous comment on the DataManager.cfc)